### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by ColyFu** [informationsea/transanno#7]
hi,

When I ran 'cargo test', it threw the attached error. (cargo version 1.64.0, CentOS7)
Looking forward to your reply, thank you.

Jun

error: failed to run custom build command for `transanno v0.3.0 (/data/Software/transanno-0.3.0/transanno)`

Caused by:
  process didn't exit successfully: `/data/Software/transanno-0.3.0/target/debug/build/transanno-6708d6f34455d257/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'byte index 7 is out of bounds of ``', library/core/src/str/mod.rs:107:9
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...

**Comment by ZReplicant** [comment]
Hello,
It seems that I the same problem:
```
error: failed to run custom build command for `transanno v0.4.2 (/mnt/c/Users/Neluloth/YandexDisk/backup/Exometry/Projects/ACMG/Materials/vcf_liftover/transanno-master/transanno)`

Caused by:
  process didn't exit successfully: `/mnt/c/Users/Neluloth/YandexDisk/backup/Exometry/Projects/ACMG/Materials/vcf_liftover/transanno-master/target/debug/build/transanno-be9f50fd4667f31d/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at 'byte index 7 is out of bounds of ``', transanno/build.rs:47:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

**Comment by kathrinsjutzeler** [comment]
> Hello, It seems that I the same problem:
> 
> ```
> error: failed to run custom build command for `transanno v0.4.2 (/mnt/c/Users/Neluloth/YandexDisk/backup/Exometry/Projects/ACMG/Materials/vcf_liftover/transanno-master/transanno)`
> 
> Caused by:
>   process didn't exit successfully: `/mnt/c/Users/Neluloth/YandexDisk/backup/Exometry/Projects/ACMG/Materials/vcf_liftover/transanno-master/target/debug/build/transanno-be9f50fd4667f31d/build-script-build` (exit status: 101)
>   --- stderr
>   thread 'main' panicked at 'byte index 7 is out of bounds of ``', transanno/build.rs:47:10
>   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> warning: build failed, waiting for other jobs to finish...
> ```

I had the same issue when I downloaded and used the zip version of the code. Using `git clone https://github.com/informationsea/transanno.git` fixed the issue. 

I built the repository, however, I couldn't use `transanno` as a command: `transanno: command not found`. As a workaround, I was able to create a chain file from a minimap file with the following line: `cargo run minimap2chain PAF_FILE.paf --output sm_chain.chain`

**Comment by informationsea** [comment]
This problem has been fixed in version 0.4.3. If needed, you can download pre-built binary files from https://github.com/informationsea/transanno/releases/latest

**Comment by kathrinsjutzeler** [comment]
Thank you so much @informationsea ! Great tool for sure!
